Live map is missing caches sporadically

User report from 2020.07.26-RC and also coincidentally seen by myself today while caching:

- Open live map (in my case OSM offline), do not hide found caches
- Have map centered to your position and keep moving (in my case riding my bike)
- After a while I reached an area, where I know, that I found a cache there but nothing was shown on the map

- I moved the map manually, no change.
- I zoomed out and in again and suddenly some missing caches appear

Remark:
I am far away from sure, whether we have a problem here.
While trying to debug, I did also see communication errors towards the geocaching servers. So maybe they stil throttle requests or have load problems.

I have a user on support mail with similar problems:
-Opening the live map
- Driving around in his car, let the map follow his position
- After first loading was OK, while moving the map only shows saved caches (and their waypoints) until he manually moves the map around.
- Then live caches are shown again and the problem starts all over if he continues moving with his car

Sys info:
```
--- System information ---
Device: SM-G950F (dreamltexx, samsung)
Android version: 9
Android build: PPR1.180610.011.G950FXXU9DTF1
c:geo version: 2020.08.04
Google Play services: enabled - 20.33.15 (100408-330018294)
Low power mode: inactive
Compass capabilities: yes
Rotation vector sensor: present
Orientation sensor: present
Magnetometer & Accelerometer sensor: present
Direction sensor used: rotation vector
Hide caches: -
Hide waypoints: -
HW acceleration: enabled (default state)
System language: de_DE
System date format: dd.MM.yy
Debug mode active: no
System internal c:geo dir: /data/user/0/cgeo.geocaching (10,7 GB free) internal
User storage c:geo dir: /storage/emulated/0/cgeo (10,7 GB free) external non-removable
Geocache data: /storage/emulated/0/Android/data/cgeo.geocaching/files/GeocacheData (10,7 GB free) external non-removable
Database: /data/user/0/cgeo.geocaching/databases/data (48,9 MB) on system internal storage
Fine location permission: granted
Write external storage permission: granted
Geocaching sites enabled:
geocaching.com: Logged in (Anmeldung OK) / PREMIUM
opencaching.de: Logged in (Anmeldung OK)
Geocaching.com date format: dd.MM.yyyy
Installed c:geo plugins: none
--- End of system information 
```

Logcat (which includes an error code 400 and an exception at the end):
```
--------- beginning of main
10-01 16:59:28.022 10017 10049 D cgeo    : [network--3] GCMap.searchPlayMapByViewport(N 48° 40,728' · E 011° 44,139',N 48° 43,665' · E 011° 54,830')
10-01 16:59:28.024 10017 17159 D cgeo    : [OkHttp] GET https://www.geocaching.com/api/proxy/web/search?box=48.72775%2C11.735653%2C48.678808%2C11.913835&take=500&asc=true&skip=0&sort=distance&origin=48.703279%2C11.824744&app=cgeo
10-01 16:59:28.024 10017 17165 D cgeo    : [OkHttp] GET https://www.opencaching.de/okapi/services/caches/shortcuts/search_and_retrieve?langpref=de%7Cen&oauth_consumer_key=J6M7x5rNz9YFJX38fE9n&oauth_nonce=c8a47e2f9b204933a53be658016e0ca1&oauth_signature_method=HMAC-SHA1&oauth_timestamp=1601564368&oauth_token=UDvnPBjPKJd3TqEsQFL4&oauth_version=1.0&retr_method=services%2Fcaches%2Fgeocaches&retr_params=%7B%22fields%22%3A%20%22code%7Cname%7Clocation%7Ctype%7Cstatus%7Cdifficulty%7Cterrain%7Csize%7Csize2%7Cdate_hidden%7Ctrackables_count%7Cis_found%22%7D&search_method=services%2Fcaches%2Fsearch%2Fbbox&search_params=%7B%22bbox%22%3A%2248.678808%7C11.735653%7C48.727750%7C11.913835%22%2C%22status%22%3A%22Available%7CTemporarily%20unavailable%22%2C%22ignored_status%22%3A%22notignored_only%22%7D&wrap=true&oauth_signature=77SbaRqWjUSaFJfFWxPGM6OV1Fw%3D
10-01 16:59:28.034 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Loading GC1ABTQ A93 Elsendorf from DB
10-01 16:59:28.035 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Loading GC30KE7 Sankt Anton from DB
10-01 16:59:28.036 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Loading GC4FPY8 S(T)EIN  oder  nicht  S(T)EIN from DB
10-01 16:59:28.038 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Loading GC6MZ8P Drei ??? Bonus 06 from DB
10-01 16:59:28.039 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Loading GC6RVCF Drei ??? #86 from DB
10-01 16:59:28.040 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Loading GC6RVCG Drei ??? #85 from DB
10-01 16:59:28.041 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Loading GC8MNEA Der Wächter from DB
10-01 16:59:28.042 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Loading OC611C A93 Elsendorf from DB
10-01 16:59:28.046 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Cache GC30KE7 for id 2 added, geoEntries: 5
10-01 16:59:28.047 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Cache GC8MNEA for id 2 added, geoEntries: 6
10-01 16:59:28.047 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Cache GC1ABTQ for id 2 not added, geoEntries: 6
10-01 16:59:28.047 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Cache GC6MZ8P for id 2 added, geoEntries: 7
10-01 16:59:28.048 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Cache OC611C for id 2 not added, geoEntries: 7
10-01 16:59:28.048 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Cache GC6RVCG for id 2 added, geoEntries: 8
10-01 16:59:28.048 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Cache GC4FPY8 for id 2 not added, geoEntries: 8
10-01 16:59:28.049 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Cache GC6RVCF for id 2 added, geoEntries: 9
10-01 16:59:28.049 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 2 synced. Codes removed: 0, new codes: 5, geoEntries: 9
10-01 16:59:28.053 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint 004FPY8 for id 0 added, geoEntries: 10
10-01 16:59:28.053 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint 014FPY8 for id 0 added, geoEntries: 11
10-01 16:59:28.054 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 0, new codes: 2, geoEntries: 11
10-01 16:59:28.297 10017 17165 D cgeo    : [OkHttp] 200 (273 ms) GET https://www.opencaching.de/okapi/services/caches/shortcuts/search_and_retrieve?langpref=de%7Cen&oauth_consumer_key=J6M7x5rNz9YFJX38fE9n&oauth_nonce=c8a47e2f9b204933a53be658016e0ca1&oauth_signature_method=HMAC-SHA1&oauth_timestamp=1601564368&oauth_token=UDvnPBjPKJd3TqEsQFL4&oauth_version=1.0&retr_method=services%2Fcaches%2Fgeocaches&retr_params=%7B%22fields%22%3A%20%22code%7Cname%7Clocation%7Ctype%7Cstatus%7Cdifficulty%7Cterrain%7Csize%7Csize2%7Cdate_hidden%7Ctrackables_count%7Cis_found%22%7D&search_method=services%2Fcaches%2Fsearch%2Fbbox&search_params=%7B%22bbox%22%3A%2248.678808%7C11.735653%7C48.727750%7C11.913835%22%2C%22status%22%3A%22Available%7CTemporarily%20unavailable%22%2C%22ignored_status%22%3A%22notignored_only%22%7D&wrap=true&oauth_signature=77SbaRqWjUSaFJfFWxPGM6OV1Fw%3D (http/1.1)
10-01 16:59:28.299 10017 10102 D cgeo    : [network--8] OC returning 2 caches from search by viewport
10-01 16:59:28.377 10017 17159 D cgeo    : [OkHttp] 200 (352 ms) GET https://www.geocaching.com/api/proxy/web/search?box=48.72775%2C11.735653%2C48.678808%2C11.913835&take=500&asc=true&skip=0&sort=distance&origin=48.703279%2C11.824744&app=cgeo (http/1.1)
10-01 16:59:28.382 10017 10049 D cgeo    : [network--3] Loading GC6HN59 Drei ??? #21 from DB
10-01 16:59:28.389 10017 10049 D cgeo    : [network--3] Loading GC6HN5G Drei ??? #84 from DB
10-01 16:59:28.392 10017 10049 D cgeo    : [network--3] Loading GC6HNTY Drei ??? Bonus #02 from DB
10-01 16:59:28.394 10017 10049 D cgeo    : [network--3] Loading GC6HNV4 Drei ??? Bonus #03 from DB
10-01 16:59:28.395 10017 10049 D cgeo    : [network--3] Loading GC6HNV5 Drei ??? #83 from DB
10-01 16:59:28.398 10017 10049 D cgeo    : [network--3] Loading GC6HNV7 Drei ??? #41 from DB
10-01 16:59:28.400 10017 10049 D cgeo    : [network--3] Loading GC6HNV9 Drei ??? #40 from DB
10-01 16:59:28.402 10017 10049 D cgeo    : [network--3] Loading GC6JMXX Drei ??? #22 from DB
10-01 16:59:28.405 10017 10049 D cgeo    : [network--3] Loading GC6JMY3 Drei ??? #23 from DB
10-01 16:59:28.407 10017 10049 D cgeo    : [network--3] Loading GC6JMYY Drei ??? #37 from DB
10-01 16:59:28.408 10017 10049 D cgeo    : [network--3] Loading GC6MZ85 Drei ??? #81 from DB
10-01 16:59:28.411 10017 10049 D cgeo    : [network--3] Loading GC6MZ87 Drei ??? #82 from DB
10-01 16:59:28.412 10017 10049 D cgeo    : [network--3] Loading GC8RHKJ 01_FiniPuk´ s Baumwipfelrunde from DB
10-01 16:59:28.415 10017 10049 D cgeo    : [network--3] Loading GC8RHMJ 02_FiniPuk´ s Baumwipfelrunde from DB
10-01 16:59:28.417 10017 10049 D cgeo    : [network--3] Loading GC8RHNN 03_FiniPuk´ s Baumwipfelrunde from DB
10-01 16:59:28.418 10017 10049 D cgeo    : [network--3] Loading GC8RHPG 04_FiniPuk´ s Baumwipfelrunde from DB
10-01 16:59:28.420 10017 10049 D cgeo    : [network--3] Loading GC8RHQ6 05_FiniPuk´ s Baumwipfelrunde from DB
10-01 16:59:28.421 10017 10049 D cgeo    : [network--3] Loading GC8RHR6 06_FiniPuk´ s Baumwipfelrunde from DB
10-01 16:59:28.423 10017 10049 D cgeo    : [network--3] Loading GC8RHW3 11_FiniPuk´ s Baumwipfelrunde from DB
10-01 16:59:28.425 10017 10049 D cgeo    : [network--3] Loading GC8RHZ3 12_FiniPuk´ s Baumwipfelrunde from DB
10-01 16:59:28.427 10017 10049 D cgeo    : [network--3] Loading GC8T33K Anna Kapelle from DB
10-01 16:59:28.429 10017 10049 D cgeo    : [network--3] DataStore.loadCaches([GC8WV9W]) returned no results
10-01 16:59:28.453 10017 10049 D cgeo    : [network--3] GCMap: returning 34 caches from search
10-01 16:59:28.455 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Filtering out 0 caches: []
10-01 16:59:28.456 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Live caches found: 36
10-01 16:59:28.457 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC8RHZ3 for id 3 added, geoEntries: 12
10-01 16:59:28.457 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC8RHMJ for id 3 added, geoEntries: 13
10-01 16:59:28.458 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC6MZ85 for id 3 added, geoEntries: 14
10-01 16:59:28.458 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC6HNV9 for id 3 added, geoEntries: 15
10-01 16:59:28.458 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC6MZ87 for id 3 added, geoEntries: 16
10-01 16:59:28.459 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC6HNV7 for id 3 added, geoEntries: 17
10-01 16:59:28.459 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC6RVCG for id 3 not added, geoEntries: 17
10-01 16:59:28.459 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC6HN59 for id 3 added, geoEntries: 18
10-01 16:59:28.459 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC6RVCF for id 3 not added, geoEntries: 18
10-01 16:59:28.459 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC30KE7 for id 3 not added, geoEntries: 18
10-01 16:59:28.459 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC6MZ8P for id 3 not added, geoEntries: 18
10-01 16:59:28.460 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC8WV9W for id 3 added, geoEntries: 19
10-01 16:59:28.460 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC8YY5R for id 3 added, geoEntries: 20
10-01 16:59:28.460 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC6JMXX for id 3 added, geoEntries: 21
10-01 16:59:28.461 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC8T33K for id 3 added, geoEntries: 22
10-01 16:59:28.461 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC6HN5G for id 3 added, geoEntries: 23
10-01 16:59:28.462 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC8RHKJ for id 3 added, geoEntries: 24
10-01 16:59:28.462 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC8RHR6 for id 3 added, geoEntries: 25
10-01 16:59:28.462 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC6JMY3 for id 3 added, geoEntries: 26
10-01 16:59:28.463 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache OC16172 for id 3 added, geoEntries: 27
10-01 16:59:28.463 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC8RHPG for id 3 added, geoEntries: 28
10-01 16:59:28.464 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC8RHW3 for id 3 added, geoEntries: 29
10-01 16:59:28.464 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC2ZPN3 for id 3 added, geoEntries: 30
10-01 16:59:28.464 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC6HNTY for id 3 added, geoEntries: 31
10-01 16:59:28.464 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC8MNEA for id 3 not added, geoEntries: 31
10-01 16:59:28.465 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC8R0AB for id 3 added, geoEntries: 32
10-01 16:59:28.465 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC6HNV5 for id 3 added, geoEntries: 33
10-01 16:59:28.466 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC6HNV4 for id 3 added, geoEntries: 34
10-01 16:59:28.466 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC6JMYY for id 3 added, geoEntries: 35
10-01 16:59:28.466 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC8RHNN for id 3 added, geoEntries: 36
10-01 16:59:28.467 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC8YY6Q for id 3 added, geoEntries: 37
10-01 16:59:28.467 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Cache GC8RHQ6 for id 3 added, geoEntries: 38
10-01 16:59:28.468 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 3 synced. Codes removed: 0, new codes: 27, geoEntries: 38
10-01 16:59:28.469 10017 17177 D cgeo    : [RxNewThreadScheduler-14] searchByViewport: cached=false, results=36, viewport=(N 48° 41,404' · E 011° 47,546',N 48° 42,788' · E 011° 48,384')
10-01 17:00:01.777 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint P08R0AB for id 0 added, geoEntries: 39
10-01 17:00:01.778 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 0, new codes: 1, geoEntries: 39
10-01 17:00:01.779 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint P08R0AB for id 0 not added, geoEntries: 39
10-01 17:01:26.775 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint 006HNV5 for id 0 added, geoEntries: 40
10-01 17:01:26.775 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 0, new codes: 1, geoEntries: 40
10-01 17:01:26.778 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint 006HNV5 for id 0 not added, geoEntries: 40
10-01 17:01:42.768 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 4, new codes: 0, geoEntries: 36
10-01 17:02:53.777 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Loading GC1R64N Maria Hilf! from DB
10-01 17:02:53.778 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Loading GC8JQVG Train from DB
10-01 17:02:53.781 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint FN8JQVG for id 0 added, geoEntries: 37
10-01 17:02:53.781 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint 021R64N for id 0 added, geoEntries: 38
10-01 17:02:53.782 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint FN8JQVG for id 0 not added, geoEntries: 38
10-01 17:02:53.782 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint P08R0AB for id 0 added, geoEntries: 39
10-01 17:02:53.783 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint P08R0AB for id 0 not added, geoEntries: 39
10-01 17:02:53.783 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint 021R64N for id 0 not added, geoEntries: 39
10-01 17:02:53.784 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint 018JQVG for id 0 added, geoEntries: 40
10-01 17:02:53.784 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint 028JQVG for id 0 added, geoEntries: 41
10-01 17:02:53.785 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint 028JQVG for id 0 not added, geoEntries: 41
10-01 17:02:53.785 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint 038JQVG for id 0 added, geoEntries: 42
10-01 17:02:53.786 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint 038JQVG for id 0 not added, geoEntries: 42
10-01 17:02:53.787 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint 011R64N for id 0 added, geoEntries: 43
10-01 17:02:53.789 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Loading GC6MZ4R Drei ??? #51 from DB
10-01 17:02:53.789 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Loading GC6MZ4R Drei ??? #51 from DB
10-01 17:02:53.792 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint 016MZ4R for id 0 added, geoEntries: 44
10-01 17:02:53.792 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint 016MZ4R for id 0 not added, geoEntries: 44
10-01 17:02:53.793 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint 011R64N for id 0 not added, geoEntries: 44
10-01 17:02:53.793 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint 048JQVG for id 0 added, geoEntries: 45
10-01 17:02:53.794 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint 048JQVG for id 0 not added, geoEntries: 45
10-01 17:02:53.794 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint 018JQVG for id 0 not added, geoEntries: 45
10-01 17:02:53.795 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 0, new codes: 3, geoEntries: 45
10-01 17:02:53.795 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 0, new codes: 6, geoEntries: 45
10-01 17:03:06.777 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 1, new codes: 0, geoEntries: 44
10-01 17:03:06.780 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 1, new codes: 0, geoEntries: 44
10-01 17:03:13.790 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint 006MZ4R for id 0 added, geoEntries: 45
10-01 17:03:13.791 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 0, new codes: 1, geoEntries: 45
10-01 17:03:13.792 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint 006MZ4R for id 0 not added, geoEntries: 45
10-01 17:03:26.781 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 2, new codes: 0, geoEntries: 43
10-01 17:03:26.785 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 2, new codes: 0, geoEntries: 43
10-01 17:04:11.785 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Loading GC7BDD0 Siegenburg - Turnerhölzl from DB
10-01 17:04:11.789 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint P07BDD0 for id 0 added, geoEntries: 44
10-01 17:04:11.793 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint P07BDD0 for id 0 not added, geoEntries: 44
10-01 17:04:11.795 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 0, new codes: 1, geoEntries: 44
10-01 17:04:17.777 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 1, new codes: 0, geoEntries: 43
10-01 17:04:17.777 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 1, new codes: 0, geoEntries: 43
10-01 17:04:33.775 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Loading GC78Q1X Siegenburger Brücken - Bonus from DB
10-01 17:04:33.776 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Loading GC78Q1X Siegenburger Brücken - Bonus from DB
10-01 17:04:33.778 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint 0178Q1X for id 0 added, geoEntries: 44
10-01 17:04:33.778 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint 0178Q1X for id 0 not added, geoEntries: 44
10-01 17:04:33.779 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 0, new codes: 1, geoEntries: 44
10-01 17:04:34.775 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint 0278Q1X for id 0 added, geoEntries: 45
10-01 17:04:34.775 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 1, new codes: 1, geoEntries: 44
10-01 17:04:34.777 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint 0278Q1X for id 0 not added, geoEntries: 44
10-01 17:04:34.777 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 1, new codes: 0, geoEntries: 44
10-01 17:04:39.777 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Loading GC34WQN Binnendünen Siegenburg from DB
10-01 17:04:39.777 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Loading GC34WQN Binnendünen Siegenburg from DB
10-01 17:04:39.779 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint P134WQN for id 0 not added, geoEntries: 45
10-01 17:04:39.779 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint P134WQN for id 0 added, geoEntries: 45
10-01 17:04:39.780 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint 0378Q1X for id 0 added, geoEntries: 46
10-01 17:04:39.780 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint 0378Q1X for id 0 not added, geoEntries: 46
10-01 17:04:39.780 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 1, new codes: 1, geoEntries: 45
10-01 17:04:39.780 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 1, new codes: 1, geoEntries: 45
10-01 17:04:40.775 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Loading GC6MZ5V Drei ??? #60 from DB
10-01 17:04:40.775 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Loading GC6MZ5V Drei ??? #60 from DB
10-01 17:04:40.777 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint PK6MZ5V for id 0 added, geoEntries: 46
10-01 17:04:40.777 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint PK6MZ5V for id 0 not added, geoEntries: 46
10-01 17:04:40.778 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 1, new codes: 0, geoEntries: 45
10-01 17:04:40.778 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 1, new codes: 1, geoEntries: 45
10-01 17:04:41.778 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Loading GC6R2VW Siegenburg Range from DB
10-01 17:04:41.778 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Loading GC6R2VW Siegenburg Range from DB
10-01 17:04:41.781 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint P06R2VW for id 0 added, geoEntries: 46
10-01 17:04:41.781 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint P06R2VW for id 0 not added, geoEntries: 46
10-01 17:04:41.781 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 0, new codes: 1, geoEntries: 46
10-01 17:04:44.773 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 1, new codes: 0, geoEntries: 45
10-01 17:04:44.774 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 1, new codes: 0, geoEntries: 45
10-01 17:04:47.773 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 1, new codes: 0, geoEntries: 44
10-01 17:04:47.775 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 1, new codes: 0, geoEntries: 44
10-01 17:04:49.774 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 1, new codes: 0, geoEntries: 43
10-01 17:04:49.775 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 1, new codes: 0, geoEntries: 43
10-01 17:04:51.773 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint 0478Q1X for id 0 added, geoEntries: 44
10-01 17:04:51.775 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint 0478Q1X for id 0 not added, geoEntries: 44
10-01 17:04:51.776 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Loading GC4ZQGB [LP] Hitlers Autobahnrelikt - Reloaded from DB
10-01 17:04:51.776 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Loading GC4ZQGB [LP] Hitlers Autobahnrelikt - Reloaded from DB
10-01 17:04:51.779 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint 004ZQGB for id 0 added, geoEntries: 45
10-01 17:04:51.779 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint 004ZQGB for id 0 not added, geoEntries: 45
10-01 17:04:51.779 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 0, new codes: 1, geoEntries: 45
10-01 17:04:51.779 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 0, new codes: 1, geoEntries: 45
10-01 17:04:52.777 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint S14ZQGB for id 0 added, geoEntries: 46
10-01 17:04:52.777 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint S14ZQGB for id 0 not added, geoEntries: 46
10-01 17:04:52.778 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 0, new codes: 1, geoEntries: 46
10-01 17:04:53.774 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint S04ZQGB for id 0 added, geoEntries: 47
10-01 17:04:53.774 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 0, new codes: 1, geoEntries: 47
10-01 17:04:53.776 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint S04ZQGB for id 0 not added, geoEntries: 47
10-01 17:04:55.775 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint PK4ZQGB for id 0 added, geoEntries: 48
10-01 17:04:55.776 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 0, new codes: 1, geoEntries: 48
10-01 17:04:55.777 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint PK4ZQGB for id 0 not added, geoEntries: 48
10-01 17:05:05.775 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint P15XTYN for id 0 added, geoEntries: 49
10-01 17:05:05.777 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint P15XTYN for id 0 not added, geoEntries: 49
10-01 17:05:05.788 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 0, new codes: 1, geoEntries: 49
10-01 17:05:10.775 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint 005XTYN for id 0 added, geoEntries: 50
10-01 17:05:10.775 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 0, new codes: 1, geoEntries: 50
10-01 17:05:10.776 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint 005XTYN for id 0 not added, geoEntries: 50
10-01 17:05:12.776 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint S15XTYN for id 0 added, geoEntries: 51
10-01 17:05:12.776 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 0, new codes: 1, geoEntries: 51
10-01 17:05:12.779 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint S15XTYN for id 0 not added, geoEntries: 51
10-01 17:05:13.775 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint S25XTYN for id 0 added, geoEntries: 52
10-01 17:05:13.776 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Layers for id 0 synced. Codes removed: 0, new codes: 1, geoEntries: 52
10-01 17:05:13.776 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint S25XTYN for id 0 not added, geoEntries: 52
10-01 17:05:53.787 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Loading GC2YVPM EILEn MIT verWEILEn from DB
10-01 17:05:53.787 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Loading GC2YVPM EILEn MIT verWEILEn from DB
10-01 17:05:53.791 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Waypoint 032YVPM for id 0 added, geoEntries: 53
10-01 17:05:53.793 10017 17177 D cgeo    : [RxNewThreadScheduler-14] Layers for id 0 synced. Codes removed: 0, new codes: 1, geoEntries: 53
10-01 17:05:53.796 10017 17176 D cgeo    : [RxNewThreadScheduler-13] Waypoint 032YVPM for id 0 not added, geoEntries: 53
10-01 17:06:18.229 10017 10017 D cgeo    : [main] NewMap: onPause
10-01 17:06:18.230 10017 10017 D cgeo    : [main] NewMap: onStop
10-01 17:06:18.230 10017 10017 D cgeo    : [main] AbtsractCacheOverlay: onDestroy overlay 2
10-01 17:06:18.231 10017 10038 D cgeo    : [looper callbacks] RotationProvider: unregistering listener
10-01 17:06:18.231 10017 10017 D cgeo    : [main] Layers for id 2 cleared, remaining geoEntries: 48
10-01 17:06:18.231 10017 10017 D cgeo    : [main] AbtsractCacheOverlay: onDestroy overlay 3
10-01 17:06:18.231 10017 10017 D cgeo    : [main] Layers for id 3 cleared, remaining geoEntries: 17
10-01 17:06:18.235 10017 10017 D cgeo    : [main] New map: onSaveInstanceState
10-01 17:06:18.236 10017 10017 D cgeo    : [main] NewMap: onDestroy
10-01 17:06:18.257 10017 10017 D cgeo    : [main] ResourceBitmapCacheMonitor.release
10-01 17:06:18.257 10017 10017 D cgeo    : [main] ResourceBitmapCacheMonitor.clearResourceBitmaps
10-01 17:06:18.329 10017 10017 D cgeo    : [main] NewMap: onCreate
10-01 17:06:18.329 10017 10017 D cgeo    : [main] ResourceBitmapCacheMonitor.addRef
10-01 17:06:18.350 10017 10017 D cgeo    : [main] Connecting brouter failed
10-01 17:06:18.358 10017 10017 D cgeo    : [main] NewMap: onStart
10-01 17:06:18.366 10017 10017 D cgeo    : [main] AbstractCacheOverlay: construct overlay 0
10-01 17:06:18.366 10017 10017 D cgeo    : [main] AbstractCacheOverlay: construct overlay 2
10-01 17:06:18.366 10017 10017 D cgeo    : [main] AbstractCacheOverlay: construct overlay 3
10-01 17:06:18.367 10017 10017 D cgeo    : [main] RestartLocationPermissionGrantedCallback.execute for 2225
10-01 17:06:18.367 10017 10038 D cgeo    : [looper callbacks] RotationProvider: registering listener
10-01 17:06:18.370 10017 10104 D cgeo    : [network--10] OC returning 0 caches from search by viewport
10-01 17:06:18.371 10017 10057 D cgeo    : [network--4] GCMap.searchPlayMapByViewport(N 48° 45,528' · E 011° 50,155',N 48° 45,528' · E 011° 50,155')
10-01 17:06:18.372 10017 10017 D cgeo    : [main] NewMap: onResume
10-01 17:06:18.375 10017 17843 D cgeo    : [OkHttp] GET https://www.geocaching.com/api/proxy/web/search?box=48.758792%2C11.835909%2C48.758792%2C11.835909&take=500&asc=true&skip=0&sort=distance&origin=48.758792%2C11.835909&app=cgeo
10-01 17:06:18.620 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC2TYZ1 Die schwarze Madonna von ? from DB
10-01 17:06:18.621 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC2YVPM EILEn MIT verWEILEn from DB
10-01 17:06:18.621 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC34WQN Binnendünen Siegenburg from DB
10-01 17:06:18.622 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC4AGCP Am Platz der verlornen Körner from DB
10-01 17:06:18.622 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC4F75T ADAC-Warte-Cache from DB
10-01 17:06:18.622 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC4KF7V Kapelle from DB
10-01 17:06:18.623 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC4ZABV Bonus - [LP] Hitlers Autobahnrelikt from DB
10-01 17:06:18.623 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC5XTYN RömerBOX from DB
10-01 17:06:18.624 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC6R2VW Siegenburg Range from DB
10-01 17:06:18.624 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC71ZAM Soldatengrab from DB
10-01 17:06:18.625 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC78Q1X Siegenburger Brücken - Bonus from DB
10-01 17:06:18.626 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC7BDD0 Siegenburg - Turnerhölzl from DB
10-01 17:06:18.627 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC7HHFM Licht am Ende .... from DB
10-01 17:06:18.628 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC7HHFW Schöner Ausblick from DB
10-01 17:06:18.629 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC86H55 Castor Fiber from DB
10-01 17:06:18.629 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC8HRB1 Mariensäule Siegenburg from DB
10-01 17:06:18.630 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC8JQVG Train from DB
10-01 17:06:18.631 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC8KQVG Beim Bankerl from DB
10-01 17:06:18.633 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Loading GC8YZHP Dorfblick    (2K 3) from DB
10-01 17:06:18.638 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC34WQN for id 2 added, geoEntries: 1
10-01 17:06:18.638 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC4AGCP for id 2 added, geoEntries: 2
10-01 17:06:18.639 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC4F75T for id 2 added, geoEntries: 3
10-01 17:06:18.639 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC2TYZ1 for id 2 added, geoEntries: 4
10-01 17:06:18.640 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC4ZABV for id 2 added, geoEntries: 5
10-01 17:06:18.641 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC2YVPM for id 2 added, geoEntries: 6
10-01 17:06:18.646 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC5XTYN for id 2 added, geoEntries: 7
10-01 17:06:18.647 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC78Q1X for id 2 added, geoEntries: 8
10-01 17:06:18.647 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC8YZHP for id 2 added, geoEntries: 9
10-01 17:06:18.648 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC86H55 for id 2 added, geoEntries: 10
10-01 17:06:18.648 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC8HRB1 for id 2 added, geoEntries: 11
10-01 17:06:18.649 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC8JQVG for id 2 added, geoEntries: 12
10-01 17:06:18.649 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC8KQVG for id 2 added, geoEntries: 13
10-01 17:06:18.650 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC6R2VW for id 2 added, geoEntries: 14
10-01 17:06:18.651 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC7HHFM for id 2 added, geoEntries: 15
10-01 17:06:18.652 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC7HHFW for id 2 added, geoEntries: 16
10-01 17:06:18.652 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC7BDD0 for id 2 added, geoEntries: 17
10-01 17:06:18.653 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC4KF7V for id 2 added, geoEntries: 18
10-01 17:06:18.654 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Cache GC71ZAM for id 2 added, geoEntries: 19
10-01 17:06:18.654 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Layers for id 2 synced. Codes removed: 0, new codes: 19, geoEntries: 19
10-01 17:06:18.657 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Waypoint 002TYZ1 for id 0 added, geoEntries: 20
10-01 17:06:18.657 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Waypoint P15XTYN for id 0 added, geoEntries: 21
10-01 17:06:18.658 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Waypoint 002YVPM for id 0 added, geoEntries: 22
10-01 17:06:18.659 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Waypoint 0378Q1X for id 0 added, geoEntries: 23
10-01 17:06:18.660 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Waypoint 018JQVG for id 0 added, geoEntries: 24
10-01 17:06:18.661 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Waypoint 032YVPM for id 0 added, geoEntries: 25
10-01 17:06:18.661 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Waypoint ST4AGCP for id 0 added, geoEntries: 26
10-01 17:06:18.662 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Waypoint 005XTYN for id 0 added, geoEntries: 27
10-01 17:06:18.662 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Waypoint S15XTYN for id 0 added, geoEntries: 28
10-01 17:06:18.663 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Waypoint PK6MZ5V for id 0 added, geoEntries: 29
10-01 17:06:18.664 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Waypoint 0478Q1X for id 0 added, geoEntries: 30
10-01 17:06:18.664 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Waypoint P07BDD0 for id 0 added, geoEntries: 31
10-01 17:06:18.665 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Waypoint S25XTYN for id 0 added, geoEntries: 32
10-01 17:06:18.665 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Waypoint PK4AGCP for id 0 added, geoEntries: 33
10-01 17:06:18.666 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Waypoint P06R2VW for id 0 added, geoEntries: 34
10-01 17:06:18.666 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Waypoint P134WQN for id 0 added, geoEntries: 35
10-01 17:06:18.667 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Waypoint 0178Q1X for id 0 added, geoEntries: 36
10-01 17:06:18.668 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Waypoint 0278Q1X for id 0 added, geoEntries: 37
10-01 17:06:18.668 10017 17840 D cgeo    : [RxNewThreadScheduler-15] Layers for id 0 synced. Codes removed: 0, new codes: 18, geoEntries: 37
10-01 17:06:19.450 10017 17843 D cgeo    : [OkHttp] 400 [Bad Request] (1075 ms) GET https://www.geocaching.com/api/proxy/web/search?box=48.758792%2C11.835909%2C48.758792%2C11.835909&take=500&asc=true&skip=0&sort=distance&origin=48.758792%2C11.835909&app=cgeo (http/1.1)
10-01 17:06:19.450 10017 10057 W cgeo    : [network--4] parallelCombineActive: swallowing error from connector cgeo.geocaching.connector.gc.GCConnector@67fe2bc
10-01 17:06:19.450 10017 10057 W cgeo    : java.lang.RuntimeException: java.io.IOException: request was not successful
10-01 17:06:19.450 10017 10057 W cgeo    : 	at io.reactivex.rxjava3.internal.util.ExceptionHelper.wrapOrThrow(ExceptionHelper.java:46)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at io.reactivex.rxjava3.internal.observers.BlockingMultiObserver.blockingGet(BlockingMultiObserver.java:94)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at io.reactivex.rxjava3.core.Single.blockingGet(Single.java:3645)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at cgeo.geocaching.connector.gc.GCWebAPI.searchMap(GCWebAPI.java:414)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at cgeo.geocaching.connector.gc.GCMap.searchPlayMapByViewport(GCMap.java:133)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at cgeo.geocaching.connector.gc.GCMap.searchByViewport(GCMap.java:105)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at cgeo.geocaching.connector.gc.GCConnector.searchByViewport(GCConnector.java:261)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at cgeo.geocaching.connector.ConnectorFactory.lambda$searchByViewport$0(ConnectorFactory.java:346)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at cgeo.geocaching.connector.-$$Lambda$ConnectorFactory$xlYIDnOeDkjd0u5VT0oIRtPq6Zk.apply(Unknown Source:4)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at cgeo.geocaching.SearchResult.lambda$null$0(SearchResult.java:327)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at cgeo.geocaching.-$$Lambda$SearchResult$S4Ub23vsnqWr5X8nr-1svPn50lg.call(Unknown Source:4)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at io.reactivex.rxjava3.internal.operators.maybe.MaybeFromCallable.subscribeActual(MaybeFromCallable.java:47)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at io.reactivex.rxjava3.core.Maybe.subscribe(Maybe.java:5317)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at io.reactivex.rxjava3.internal.operators.maybe.MaybeSubscribeOn$SubscribeTask.run(MaybeSubscribeOn.java:54)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:41)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:28)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at java.lang.Thread.run(Thread.java:764)
10-01 17:06:19.450 10017 10057 W cgeo    : Caused by: java.io.IOException: request was not successful
10-01 17:06:19.450 10017 10057 W cgeo    : 	at cgeo.geocaching.network.Network.lambda$static$6(Network.java:621)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at cgeo.geocaching.network.-$$Lambda$Network$S1n6XPPgWcknnAjXJlYG-giT7W4.apply(Unknown Source:2)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at io.reactivex.rxjava3.internal.operators.single.SingleFlatMap$SingleFlatMapCallback.onSuccess(SingleFlatMap.java:77)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at io.reactivex.rxjava3.internal.operators.single.SingleCreate$Emitter.onSuccess(SingleCreate.java:68)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at cgeo.geocaching.utils.RxOkHttpUtils$1.onResponse(RxOkHttpUtils.java:48)
10-01 17:06:19.450 10017 10057 W cgeo    : 	at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:504)
10-01 17:06:19.450 10017 10057 W cgeo    : 	... 3 more
10-01 17:06:19.451 10017 17841 D cgeo    : [RxNewThreadScheduler-16] Filtering out 0 caches: []
10-01 17:06:19.451 10017 17841 D cgeo    : [RxNewThreadScheduler-16] Live caches found: 0
10-01 17:06:19.453 10017 17841 D cgeo    : [RxNewThreadScheduler-16] searchByViewport: cached=false, results=0, viewport=null
10-01 17:06:19.454 10017 10100 D cgeo    : [network--7] GCMap.searchPlayMapByViewport(N 48° 42,411' · E 011° 47,373',N 48° 48,644' · E 011° 52,935')
10-01 17:06:19.455 10017 17843 D cgeo    : [OkHttp] GET https://www.geocaching.com/api/proxy/web/search?box=48.810732%2C11.789557%2C48.706848%2C11.882257&take=500&asc=true&skip=0&sort=distance&origin=48.75879%2C11.835907&app=cgeo
10-01 17:06:19.458 10017 17857 D cgeo    : [OkHttp] GET https://www.opencaching.de/okapi/services/caches/shortcuts/search_and_retrieve?langpref=de%7Cen&oauth_consumer_key=J6M7x5rNz9YFJX38fE9n&oauth_nonce=bb61e821be8f1aa1ad10aaddfb920d6b&oauth_signature_method=HMAC-SHA1&oauth_timestamp=1601564779&oauth_token=UDvnPBjPKJd3TqEsQFL4&oauth_version=1.0&retr_method=services%2Fcaches%2Fgeocaches&retr_params=%7B%22fields%22%3A%20%22code%7Cname%7Clocation%7Ctype%7Cstatus%7Cdifficulty%7Cterrain%7Csize%7Csize2%7Cdate_hidden%7Ctrackables_count%7Cis_found%22%7D&search_method=services%2Fcaches%2Fsearch%2Fbbox&search_params=%7B%22bbox%22%3A%2248.706848%7C11.789557%7C48.810732%7C11.882257%22%2C%22status%22%3A%22Available%7CTemporarily%20unavailable%22%2C%22ignored_status%22%3A%22notignored_only%22%7D&wrap=true&oauth_signature=tN1sX%2B3TPlel9uh6Kdt6ayYUL64%3D
10-01 17:06:19.806 10017 17857 D cgeo    : [OkHttp] 200 (348 ms) GET https://www.opencaching.de/okapi/services/caches/shortcuts/search_and_retrieve?langpref=de%7Cen&oauth_consumer_key=J6M7x5rNz9YFJX38fE9n&oauth_nonce=bb61e821be8f1aa1ad10aaddfb920d6b&oauth_signature_method=HMAC-SHA1&oauth_timestamp=1601564779&oauth_token=UDvnPBjPKJd3TqEsQFL4&oauth_version=1.0&retr_method=services%2Fcaches%2Fgeocaches&retr_params=%7B%22fields%22%3A%20%22code%7Cname%7Clocation%7Ctype%7Cstatus%7Cdifficulty%7Cterrain%7Csize%7Csize2%7Cdate_hidden%7Ctrackables_count%7Cis_found%22%7D&search_method=services%2Fcaches%2Fsearch%2Fbbox&search_params=%7B%22bbox%22%3A%2248.706848%7C11.789557%7C48.810732%7C11.882257%22%2C%22status%22%3A%22Available%7CTemporarily%20unavailable%22%2C%22ignored_status%22%3A%22notignored_only%22%7D&wrap=true&oauth_signature=tN1sX%2B3TPlel9uh6Kdt6ayYUL64%3D (http/1.1)
10-01 17:06:19.808 10017 10048 D cgeo    : [network--2] Loading OC611B A93 Siegenburg from DB
10-01 17:06:19.818 10017 10048 D cgeo    : [network--2] DataStore.loadCaches([OC14AEF]) returned no results
10-01 17:06:19.820 10017 10048 D cgeo    : [network--2] DataStore.loadCaches([OCC3E3]) returned no results
10-01 17:06:19.822 10017 10048 D cgeo    : [network--2] DataStore.loadCaches([OC137AE]) returned no results
10-01 17:06:19.824 10017 10048 D cgeo    : [network--2] DataStore.loadCaches([OC34C7]) returned no results
10-01 17:06:19.826 10017 10048 D cgeo    : [network--2] DataStore.loadCaches([OC163BA]) returned no results
10-01 17:06:19.827 10017 10048 D cgeo    : [network--2] DataStore.loadCaches([OC163B9]) returned no results
10-01 17:06:19.829 10017 10048 D cgeo    : [network--2] DataStore.loadCaches([OCC75C]) returned no results
10-01 17:06:19.831 10017 10017 D cgeo    : [main] NewMap: onPause
10-01 17:06:19.831 10017 10048 D cgeo    : [network--2] OC returning 10 caches from search by viewport
10-01 17:06:20.214 10017 10017 D cgeo    : [main] NewMap: onStop
10-01 17:06:20.214 10017 10038 D cgeo    : [looper callbacks] RotationProvider: unregistering listener
10-01 17:06:20.215 10017 10017 D cgeo    : [main] AbtsractCacheOverlay: onDestroy overlay 2
10-01 17:06:20.215 10017 10017 D cgeo    : [main] Layers for id 2 cleared, remaining geoEntries: 18
10-01 17:06:20.215 10017 10017 D cgeo    : [main] AbtsractCacheOverlay: onDestroy overlay 3
10-01 17:06:20.216 10017 10017 D cgeo    : [main] Layers for id 3 cleared, remaining geoEntries: 18
10-01 17:06:20.223 10017 10017 D cgeo    : [main] NewMap: onDestroy
10-01 17:06:20.227 10017 17843 W cgeo    : [OkHttp] Failure (772 ms) GET https://www.geocaching.com/api/proxy/web/search?box=48.810732%2C11.789557%2C48.706848%2C11.882257&take=500&asc=true&skip=0&sort=distance&origin=48.75879%2C11.835907&app=cgeo (java.net.SocketException: Socket closed)
10-01 17:06:20.227 10017 10100 W cgeo    : [network--7] parallelCombineActive: swallowing error from connector cgeo.geocaching.connector.gc.GCConnector@67fe2bc
10-01 17:06:20.227 10017 10100 W cgeo    : java.lang.RuntimeException: java.lang.InterruptedException
10-01 17:06:20.227 10017 10100 W cgeo    : 	at io.reactivex.rxjava3.internal.util.ExceptionHelper.wrapOrThrow(ExceptionHelper.java:46)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at io.reactivex.rxjava3.internal.observers.BlockingMultiObserver.blockingGet(BlockingMultiObserver.java:89)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at io.reactivex.rxjava3.core.Single.blockingGet(Single.java:3645)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at cgeo.geocaching.connector.gc.GCWebAPI.searchMap(GCWebAPI.java:414)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at cgeo.geocaching.connector.gc.GCMap.searchPlayMapByViewport(GCMap.java:133)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at cgeo.geocaching.connector.gc.GCMap.searchByViewport(GCMap.java:105)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at cgeo.geocaching.connector.gc.GCConnector.searchByViewport(GCConnector.java:261)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at cgeo.geocaching.connector.ConnectorFactory.lambda$searchByViewport$0(ConnectorFactory.java:346)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at cgeo.geocaching.connector.-$$Lambda$ConnectorFactory$xlYIDnOeDkjd0u5VT0oIRtPq6Zk.apply(Unknown Source:4)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at cgeo.geocaching.SearchResult.lambda$null$0(SearchResult.java:327)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at cgeo.geocaching.-$$Lambda$SearchResult$S4Ub23vsnqWr5X8nr-1svPn50lg.call(Unknown Source:4)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at io.reactivex.rxjava3.internal.operators.maybe.MaybeFromCallable.subscribeActual(MaybeFromCallable.java:47)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at io.reactivex.rxjava3.core.Maybe.subscribe(Maybe.java:5317)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at io.reactivex.rxjava3.internal.operators.maybe.MaybeSubscribeOn$SubscribeTask.run(MaybeSubscribeOn.java:54)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:41)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:28)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at java.lang.Thread.run(Thread.java:764)
10-01 17:06:20.227 10017 10100 W cgeo    : Caused by: java.lang.InterruptedException
10-01 17:06:20.227 10017 10100 W cgeo    : 	at java.util.concurrent.locks.AbstractQueuedSynchronizer.doAcquireSharedInterruptibly(AbstractQueuedSynchronizer.java:1022)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at java.util.concurrent.locks.AbstractQueuedSynchronizer.acquireSharedInterruptibly(AbstractQueuedSynchronizer.java:1328)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at java.util.concurrent.CountDownLatch.await(CountDownLatch.java:232)
10-01 17:06:20.227 10017 10100 W cgeo    : 	at io.reactivex.rxjava3.internal.observers.BlockingMultiObserver.blockingGet(BlockingMultiObserver.java:86)
10-01 17:06:20.227 10017 10100 W cgeo    : 	... 18 more
10-01 17:06:20.227 10017 17841 W cgeo    : [RxNewThreadScheduler-16] LiveCachesOverlay.startLoadtimer.start
10-01 17:06:20.227 10017 17841 W cgeo    : java.lang.RuntimeException: java.lang.InterruptedException
10-01 17:06:20.227 10017 17841 W cgeo    : 	at io.reactivex.rxjava3.internal.util.ExceptionHelper.wrapOrThrow(ExceptionHelper.java:46)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at io.reactivex.rxjava3.internal.observers.BlockingMultiObserver.blockingGet(BlockingMultiObserver.java:89)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at io.reactivex.rxjava3.core.Single.blockingGet(Single.java:3645)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at cgeo.geocaching.SearchResult.parallelCombineActive(SearchResult.java:336)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at cgeo.geocaching.connector.ConnectorFactory.searchByViewport(ConnectorFactory.java:346)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at cgeo.geocaching.maps.mapsforge.v6.caches.LiveCachesOverlay.download(LiveCachesOverlay.java:104)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at cgeo.geocaching.maps.mapsforge.v6.caches.LiveCachesOverlay.access$100(LiveCachesOverlay.java:28)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at cgeo.geocaching.maps.mapsforge.v6.caches.LiveCachesOverlay$LoadTimerAction.run(LiveCachesOverlay.java:83)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at io.reactivex.rxjava3.core.Scheduler$PeriodicDirectTask.run(Scheduler.java:566)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at io.reactivex.rxjava3.core.Scheduler$Worker$PeriodicTask.run(Scheduler.java:513)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at io.reactivex.rxjava3.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:65)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at io.reactivex.rxjava3.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:56)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at java.lang.Thread.run(Thread.java:764)
10-01 17:06:20.227 10017 17841 W cgeo    : Caused by: java.lang.InterruptedException
10-01 17:06:20.227 10017 17841 W cgeo    : 	at java.util.concurrent.locks.AbstractQueuedSynchronizer.doAcquireSharedInterruptibly(AbstractQueuedSynchronizer.java:1022)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at java.util.concurrent.locks.AbstractQueuedSynchronizer.acquireSharedInterruptibly(AbstractQueuedSynchronizer.java:1328)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at java.util.concurrent.CountDownLatch.await(CountDownLatch.java:232)
10-01 17:06:20.227 10017 17841 W cgeo    : 	at io.reactivex.rxjava3.internal.observers.BlockingMultiObserver.blockingGet(BlockingMultiObserver.java:86)
10-01 17:06:20.227 10017 17841 W cgeo    : 	... 15 more
10-01 17:06:20.233 10017 10017 D cgeo    : [main] ResourceBitmapCacheMonitor.release
10-01 17:06:20.233 10017 10017 D cgeo    : [main] ResourceBitmapCacheMonitor.clearResourceBitmaps
10-01 17:06:25.088 10017 10017 I cgeo    : [main] Added '/storage/emulated/0/Android/data/cgeo.geocaching/files' as available external dir
10-01 17:06:27.506 10017 10038 D cgeo    : [looper callbacks] LocationProvider: stopping location requests
```

--- a/main/src/cgeo/geocaching/maps/mapsforge/v6/caches/LiveCachesOverlay.java
+++ b/main/src/cgeo/geocaching/maps/mapsforge/v6/caches/LiveCachesOverlay.java
@@ -29,7 +29,6 @@ public class LiveCachesOverlay extends AbstractCachesOverlay {
 
     private final Disposable timer;
     private boolean downloading = false;
-    public final long loadThreadRun = -1;
 
     private SearchResult lastSearchResult = null;
     private Viewport lastViewport = null;
@@ -49,7 +48,9 @@ public class LiveCachesOverlay extends AbstractCachesOverlay {
         @NonNull
         private final WeakReference<LiveCachesOverlay> overlayRef;
         private int previousZoom = -100;
-        private Viewport previousViewport;
+        private Viewport previousCycleViewport; //viewport on last timer cycle
+        private Viewport previousMoveViewport; //viewport on last move
+        private long lastMovedTimestamp = -1;
 
         LoadTimerAction(@NonNull final LiveCachesOverlay overlay) {
             this.overlayRef = new WeakReference<>(overlay);
@@ -70,22 +71,24 @@ public class LiveCachesOverlay extends AbstractCachesOverlay {
 
                 // check if map moved or zoomed
                 //TODO Portree Use Rectangle inside with bigger search window. That will stop reloading on every move
-                final boolean moved = overlay.isInvalidated() || previousViewport == null || zoomNow != previousZoom ||
-                        mapMoved(previousViewport, viewportNow);
+                final boolean moved = overlay.isInvalidated() || previousCycleViewport == null || zoomNow != previousZoom ||
+                        previousMoveViewport == null || mapMoved(previousMoveViewport, viewportNow);
 
                 // save new values
                 if (moved) {
                     final long currentTime = System.currentTimeMillis();
 
-                    if (1000 < (currentTime - overlay.loadThreadRun)) {
+                    if (1000 < (currentTime - lastMovedTimestamp)) {
                         overlay.downloading = true;
                         previousZoom = zoomNow;
                         overlay.download();
+                        previousMoveViewport = viewportNow;
+                        lastMovedTimestamp = System.currentTimeMillis();
                     }
-                } else if (!previousViewport.equals(viewportNow)) {
+                } else if (!previousCycleViewport.equals(viewportNow)) {
                     overlay.updateTitle();
                 }
-                previousViewport = viewportNow;
+                previousCycleViewport = viewportNow;
             } catch (final Exception e) {
                 Log.w("LiveCachesOverlay.startLoadtimer.start", e);
             } finally {

--- a/main/src/cgeo/geocaching/network/Network.java
+++ b/main/src/cgeo/geocaching/network/Network.java
@@ -618,7 +618,7 @@ public final class Network {
                 response.close();
             }
         }
-        return Single.error(new IOException("request was not successful"));
+        return Single.error(new IOException("request was not successful: " + response));
     };
 
     /**

